docs/adr/funding-guidelines.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Funding Use Guidelines
+description: Guidelines for spending project funding
+labels: ADR
+---
+
+# ADR [Number]: Funding Use Guidelines
+
+## Status
+
+Proposed
+
+## Submitters
+
+- @ctcpip
+- @UlisesGascon
+- @wesleytodd
+
+## Decision Owners
+
+- @expressjs/express-tc
+
+## Context
+
+We have an Open Collective but we originally decided to wait on setting guidelines for how we will spend the funds. Now that we have landed PRs to include the `funding` field in `package.json` and also a link in the footer of the website, we wanted to have some loose guidelines so contributors know a little about how we intend to use financial support for the project.
+
+## Decision
+
+Funds are not automatically distributed.
+
+TC members may request funds to be distributed for the following purposes:
+
+- Living expenses (food, shelter, medicine/healthcare, transportation, utilities)
+  - The requester need not specify any detail other than the amount being requested.
+- Travel (hotel and airfare) to events related to Express or the greater Node/JS ecosystem
```

Comment:
Shouldn't we specify that they must be of legal age according to their locality?

Reply:
Great catch! Following the CPC example: https://github.com/openjs-foundation/cross-project-council/blob/a177a05a469ca101ac158d22aa4fb4b1af8725f1/community-fund/COMMUNITY_FUND_POLICY.md?plain=1#L9

Reply:
Added a siggestion for this https://github.com/expressjs/discussions/pull/299/files#r1830828177